tests/migrations/compose.yaml

```diff
@@ -29,12 +29,13 @@ services:
     command:
     - -c
     - |
+      export DRACONCTL_MIGRATIONS_PATH='/etc/dracon/migrations/enrichment'
       apt-get update && apt-get install -y jq &&
       echo "------------------------------ Starting tests ------------------------------" &&
       /bin/draconctl migrations inspect --url postgres://postgres:password@postgres:5432?sslmode=disable --json /etc/dracon/migrations/enrichment | tee | jq -eM '. | select(.Version == 0)' &&
       echo "------------------------------ Applying migrations ------------------------------" &&
-      /bin/draconctl migrations apply --url postgres://postgres:password@postgres:5432?sslmode=disable /etc/dracon/migrations/enrichment &&
-      /bin/draconctl migrations inspect --url postgres://postgres:password@postgres:5432?sslmode=disable --json /etc/dracon/migrations/enrichment | tee | jq -eM '. | select(.Version > 0)' &&
+      /bin/draconctl migrations apply --url postgres://postgres:password@postgres:5432?sslmode=disable --migrations-path /etc/dracon/migrations/enrichment &&
```

Comment:
I don't think the flag is needed any more, since the command should pick up the correct path from the environment variable?